skin/src/main/java/org/researchstack/skin/model/User.java

```diff
@@ -7,26 +7,60 @@
  */
 public class User implements Serializable
 {
+    private String fullname;
+
     private String name;
 
+    private String password;
+
     private String email;
 
     private String birthDate;
 
-    public User()
-    {
+    public User() {
+    }
+
+    public String getFullname() {
+        return fullname;
+    }
+
+    public void setFullname(String fullname) {
+        this.fullname = fullname;
     }
 
+    /**
+     * Retrieves the username, as registered on the server
+     */
     public String getName()
     {
         return name;
     }
 
+    /**
+     * Sets the username, as registered on the server.
+     */
     public void setName(String name)
     {
         this.name = name;
     }
 
+    /**
+     * Gets the password of the user, as registered on the server.
+     */
+    public String getPassword()
```

Comment:
I am using it as it is being used in MoleMapper. But I need to keep more information than the one used in Mole Mapper. I am OK to remove this class from RS and leave it to the app.